x-pack/plugins/canvas/types/canvas.ts

```diff
@@ -59,6 +64,7 @@ export interface CanvasWorkpad {
   page: number;
   pages: CanvasPage[];
   width: number;
+  sidebar: Sidebar;
```

Comment:
This should not be added as a part of the `CanvasWorkpad` type.  And the `sidebar: {}` should be removed from all of the templates.

Reply:
Done.